browser/js/common/directives/navbar/navbar.html

```diff
@@ -1,35 +1,63 @@
-<nav class="navbar navbar-static-top" ng-hide="hide">
+<nav class="navbar navbar-static-top" ng-hide="hide" ng-cloak>
     <div class="container nav-container">
         <!-- <div class="navbar-header">
             <a class="navbar-brand">LASSO</a>
         </div> -->
-<!--         <div class="navbar-header">
- -->        <button ng-init="navCollapsed = true" ng-click="navCollapsed = !navCollapsed" class="navbar-toggle collapsed" data-toggle="uib-collapse" data-target="#burger">
+        <!--         <div class="navbar-header">
+ -->
+        <button ng-init="navCollapsed = true" ng-click="navCollapsed = !navCollapsed" class="navbar-toggle collapsed" data-toggle="uib-collapse" data-target="#burger">
```

Comment:
I'd recommend against `ng-init`. Any logic in an `ng-init` can usually be safely moved to the containing controller. Mainly it's just expected to find `$scope` management logic in the controller and not the view.